draft-ietf-rats-architecture.md

```diff
@@ -273,12 +273,24 @@ plans to support in the TEEP architecture {{?I-D.ietf-teep-architecture}}.
 
 # Two Types of Environments
 
-An Attester consists of at least one Attesting Environment and Attested Environment.
-In some implementations, the Attesting and Attested Environments might be combined.
-Other implementations might have multiple Attesting and Attested Environments.
-
-    <this section can include Two Types of Environments content from draft-birkholz-rats-architecture
-    but can we find a better name? also this could be a subsection of something else?>
+An Attester consists of at least one Attesting Environment and at least one
+Target Environment. In some implementations, the Attesting and Target Environments
```

Comment:
Just to make sure: Target Environment is a proposal to replace Attesting Environment (was calling it Execution Environment in the other PR before. Sry, my mistake), yes? Consequently this would result in a change of the term in the corresponding section. That is probably a separate issue?

Reply:
Yes, that was the suggestion on the list.